lms/static/sass/course/_dates.scss

```diff
@@ -0,0 +1,120 @@
+.date-wrapper {
```

Comment:
So we have like 3 different css systems in platform (v1 - pure sass, v2 - pattern library, v3 - bootstrap). This looks like pure sass? Which is fine? v1 and v3 are both "supported" but we are obviously trying to not write more v2.